docs/plugins.md

```diff
@@ -782,6 +782,56 @@ Possible config options:
 
 -   `config` **any** 
 
+## tryTo
+
+Adds global `tryTo` function inside of which all failed steps won't fail a test but will return true/false.
+
+```js
+const result = await tryTo(() => I.see('Welcome'));
+
+// if user is on page, result => true
+// if user is on page, result => false
+```
+
+Disables retryFailedStep plugin for steps inside a block;
```

Comment:
```suggestion
Disables retryFailedStep plugin for steps inside a block.
```

Reply:
@DavertMik Is this info about that retryFailedStep plugin is disabled for steps inside "tryTo"?

Reply:
Yes, you are right! Can you propose better wording here?